templates/firstaide/create_post.html

```diff
@@ -102,7 +102,7 @@ <h2 class="text-center inverted-font">New Card</h2>
     <br>
     <form class="form-horizontal" method="post">
         {% csrf_token %}
-        
+
```

Comment:
Same as above. Please remove this change.

Reply:
Removing this will cluster things in this section of the code